.github/workflows/main.yml

```diff
@@ -31,14 +31,8 @@ jobs:
           node-version: 18
       - run: cargo fmt --all --check
       - run: cargo fmt --all --check --manifest-path examples/Cargo.toml
-      - run: cargo fmt --all --check --manifest-path templates/bonsai/Cargo.toml
       - run: cargo sort --workspace --check
       - run: cargo sort --workspace --check examples
-      - run: cargo sort --workspace --check templates/bonsai
-      - run: npm install
-        working-directory: templates/bonsai
-      - run: npm run ci
-        working-directory: templates/bonsai
```

Comment:
Hopefully all these changes are temporary?

Reply:
These were used for testing the old bonsai template. They won't be needed to test the new one I believe. I need the foundry installation action pulled in so I can use it to create the test.